analyzer/src/funTest/assets/projects/synthetic/maven-wagon-expected-output.yml

```diff
@@ -30,12 +30,12 @@ project:
           \ ProjectBuildingException: Error resolving project artifact: Could not\
           \ transfer artifact com.example:ftp-artifact:pom:1.0.0 from/to ftp-repository\
           \ (ftp://localhost:40123): Password not specified for repository ftp-repository\
-          \ for project com.example:ftp-artifact:pom:1.0.0\nCaused by: ArtifactResolutionException:\
+          \ for project com.example:ftp-artifact:pom:1.0.0\n    Cause: ArtifactResolutionException:\
```

Comment:
The formatting / wording changes make this a breaking change, as resolutions might no longer match.
I think we should stay compatible as drop the cosmetic changes.

Reply:
Unfortunately, we cannot stay fully compatible as causes of suppressed exception also also included now in the message, which is the actual purpose of this change. However, I've changed the formatting a bit again to reduce the likeliness of breaking changes.